CardScan/Classes/NMSUtil.swift

```diff
@@ -0,0 +1,71 @@
+//
+//  NMSUtil.swift
+//  CardScan
+//
+//  Created by Zain on 8/6/19.
+//
+
+import Foundation
+struct NMSUtils{
+    static func IOUOf(currentBox: [Float], nextBox: [Float]) -> Float{
```

Comment:
is it possible to use some sort of apple structure (like CGRect) for this?

Reply:
This comment is optional, this code is fine as is for now

Reply:
ideally I'd love to see this implemented using an extension to a CGRect that is something like `iou(rect: CGRect)`